lib/repository/repo.dart

```diff
@@ -1,29 +1,27 @@
-
-
 import 'package:flutter/material.dart';
-import 'package:mixeditems/widgets/expendablelist_section.dart';
+import 'package:mixeditems/widgets/expandablelist.dart';
 import 'package:mixeditems/widgets/iconlist_section.dart';
 import 'package:mixeditems/widgets/iconlistitem_section.dart';
 import 'package:mixeditems/widgets/information_section.dart';
 import 'package:mixeditems/widgets/question_section.dart';
 import 'package:mixeditems/widgets/switch_section.dart';
 
 import '../models/listitem.dart';
-class Repo {
+import '../widgets/expendablelistitem_section.dart';
 
-
-  List<ListItem> getListItems(){
+class Repo {
+  List<ListItem> getListItems() {
     return [
-    SwitchItem('Latest Location', true),
-    IconListHead('Latest Access'),
-    IconListItem('Fit', '5 minutes ago', Icons.fitness_center),
-    IconListItem('Emergency Information', '8 minutes ago', Icons.warning),
-    IconListItem('Maps', '8 minutes ago', Icons.pin_drop),
-    ExpandableListSection('See all'),
-    QuestionTextSection('Welche Apps dürfen den Standort ermitteln?',
-    '28 von 90 Apps haben Zugriff auf den Standort'),
-    InfoSection('Location Service',
-    "You can allow your phone to use GPS, Wi-Fi networks, and cellular networks to determine your approximate location. Applications that have permission can use this information to provide location-based services, such as checking in for a flight, viewing traffic information, finding nearby restaurants, or tagging your photos with location information.")
+      SwitchItem('Latest Location', true),
+      IconListHead('Latest Access'),
+      IconListItem('Fit', '5 minutes ago', Icons.fitness_center),
+      IconListItem('Emergency Information', '8 minutes ago', Icons.warning),
+      IconListItem('Maps', '8 minutes ago', Icons.pin_drop),
+      ExpandableListSection('See all'),
```

Comment:
Better have two different Expandable Items here instead of two children in the section. One Expandable per Expanding view makes more sense.

Reply:
I should make IconList as Expandable one (but always expanded) right?

Reply:
At the time of writing you had two expandable views inside the ExpandableListSection. You could instead make two sections with one expandable view each.

Reply:
I understand. For now, I have one Section with column of ListTiles and one Section with column of ExpansionTiles.  You want me to try make only one for example ListSection contains two Widget List of each of them?